.travis.yml

```diff
@@ -1,5 +1,9 @@
 sudo: false
 language: node_js
+cache:
+  yarn: true
+  directories:
+    - node_modules
 matrix:
   include:
     - node_js: "7"
```

Comment:
We may want to consider adding a job for node 8, since that's the current latest version and will become the LTS version in a few months.